src/Auth0.php

```diff
@@ -258,6 +258,36 @@ public function login($state = null, $connection = null)
         exit;
     }
 
+    public function clientCredentials()
+    {
+        $params = [];
+        if ($this->audience) {
+            $params['audience'] = $this->audience;
+        }
+        if ($this->scope) {
+            $params['scope'] = $this->scope;
+        }
+
+        $params['response_mode'] = $this->response_mode;
+
+        $response = $this->authentication->client_credentials($params);
+
+        $access_token = (isset($response['access_token'])) ? $response['access_token'] : false;
+        $refresh_token = false;
+        $id_token = false;
+
+        if (!$access_token) {
+            throw new ApiException('Unable to retrieve access token. Invalid credentials.');
+        }
+
+        $this->setAccessToken($access_token);
+        $this->setIdToken($id_token);
```

Comment:
Where does it get the ID token from?

Reply:
I'm just setting the id_token to false here so that if there is an existing value it is deleted.